fe/src/components/base/Search.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div class="flex space-x-4 p-2 rounded-md bg-gray-200">
+    <Icon name="charm:search" class="my-auto text-disabled" />
+    <input placeholder="Search" class="bg-gray-200 focus:bg-white" />
+  </div>
+</template>
+
+<script setup></script>
+
+<style scoped></style>
```

Comment:
We're using tailwindcss for styles. <style> tag is no longer used. Please remove.
```suggestion
```